### ISSUE TITLE
question on no-op statement in blueprint hook

### ISSUE THREAD

**Issue by tjwebb** [balderdashy/sails#3059]
https://github.com/balderdashy/sails/blob/master/lib/hooks/blueprints/index.js#L427

@sgress454 @mikermcneil what does this do?

**Comment by sgress454** [comment]
Beats me.  Maybe it was just for clarity, because `BlueprintController` is created at the very top of the script.

**Comment by sailsbot** [comment]
Thanks for posting, @tjwebb.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
